ios/FluentUI/Vnext/Avatar/AvatarGroup.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public enum MSFAvatarGroupStyle: Int, CaseIterable {
+    case stack
+    case pile
+}
+
+/// Properties that make up AvatarGroup content
+@objc public class MSFAvatarGroupState: NSObject, ObservableObject {
```

Comment:
Can we have the a public protocol named MSFAvatarGroupState and switch this to an internal class called MSFAvatarGroupStateImpl?
We should apply this pattern to avoid exposing our state implementation publicly.